src/domain/accounts/entities/__tests__/account.builder.ts

```diff
@@ -5,8 +5,8 @@ import { getAddress } from 'viem';
 
 export function accountBuilder(): IBuilder<Account> {
   return new Builder<Account>()
-    .with('id', faker.number.int({ max: Number.MAX_SAFE_INTEGER }))
-    .with('group_id', faker.number.int({ max: Number.MAX_SAFE_INTEGER }))
+    .with('id', faker.number.int({ max: 1_000_000 }))
+    .with('group_id', faker.number.int({ max: 1_000_000 }))
```

Comment:
This is an arbitrary value, I think this can be extracted to a constants file, under a descriptive name like `DB_MAX_SAFE_INTEGER` or something similar, wdyt?

Reply:
Sounds good to me. (I checked the default `max` to make sure we're not increasing it and it's `Number.MAX_SAFE_INTEGER`.)